client/src/js/app.js

```diff
@@ -719,6 +719,10 @@ function bhimaconfig($routeProvider) {
     controller : 'configureBilan',
     templateUrl : 'partials/reports_proposed/bilan/bilan.html'
   })
+  .when('/reports/debitor_group_report/', {
+    controller : 'configureDebitorGroupReport as configureDebitorGroupReportCtrl',
```

Comment:
It might be nicer to use a shorter name that is easier to read in the 'as' statement.